frontends/beta-frontend/libs/fetcher/index.ts

```diff
@@ -43,9 +43,11 @@ export const connector = async (url: string, method: string, body?: any) => {
     },
     body: JSON.stringify(body),
   });
-  if (response.ok) {
+  if (response.ok && url.startsWith("http://localhost:8082")) {
```

Comment:
This obviously can not stay like that, I think I do not need to explain why :D 
API does not return `json` that is why the error appears. So I see two possible solutions - ask BE to change return value to valid `json` (empty but still valid) or check what content-type header they expected, set this in our request and refactor connector to support more than only ` "application/json"` content-type. This may be for example `text/plain` so you can not use `response.json()` there. Check this but if you need help I can help you